.github/workflows/grid-proxy-build.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
   pull_request:
     types: [opened, reopened, review_requested]
+  schedule:
+    - cron: "25 5 * * *"
```

Comment:
I think this workflow publish the docker image too we don't want it published nightly, right?

Reply:
nvm, this workflow doesn't push the image